### DISCUSSION TITLE
Remember ui state when in profile and downloading songs

### DISCUSSION THREAD

**Discussion by bytenekos** [ppy/osu#25498] [Ideas] [answered]
https://github.com/ppy/osu/assets/66184829/37bf7f6f-5822-4f57-a8e7-a6b85ccf4389

Would be nice if osu remembered where you were when going in between menus. Here I would've expected that it would bring me back to the profile menu after closing the song menu, not return me back all the way.

**Comment by bytenekos** [comment]
Upon further testing, it seems like it only half remembers? Or whatever logic is being used isn't being used for the profile specifically or something

https://github.com/ppy/osu/assets/66184829/705d6118-ed60-4bf5-90ff-ab7f33845005

**Reply by Theighlin** [reply, marked as answer]
#5063
